src/content/docs/properties/work-with-properties/manage-properties.mdx

```diff
@@ -7,53 +7,54 @@ relatedArticles:
   - f297a051-485f-4d7e-ae70-58063e96b60a
   - 8b1b2198-e919-484a-846b-e89422178707
   - b760f3bc-1f80-4edb-b7b2-94bc9a5ad081
+  - 0873e24c-1d8c-4bc1-bb27-df34cb9d1cf5
 app_context:
   - m: properties
     s: properties
 ---
 
-For properties to be available in a user or organization record, they first need to exist in your business. This topic describes how to create and edit properties.
+For properties to be available in an application, user or organization record, they first need to exist in your business. This topic describes how to create and edit properties.
 
-You can also create and manage properties through the [Kinde Management API](https://kinde.com/api/docs/#kinde-management-api-properties).
+Follow the procedure below to:
+
+- add properties for users
+- add properties for organizations
+- add properties for applications (e.g. M2M applications)
```

Comment:
@clairekinde11 not sure if we need `(e.g. M2M applications)` - we could say `(including M2M applications)` but not sure if needed